server/services/healthCheck.ts

```diff
@@ -46,11 +46,12 @@ function addAppInfo(result: HealthCheckResult, applicationInfo: ApplicationInfo)
 }
 
 function gatherCheckInfo(aggregateStatus: Record<string, unknown>, currentStatus: HealthCheckStatus) {
-  return { ...aggregateStatus, [currentStatus.name]: currentStatus.message }
+  return { ...aggregateStatus, [currentStatus.name]: { status: currentStatus.status, details: currentStatus.message } }
 }
 
 const apiChecks = [
   service('hmppsAuth', `${config.apis.hmppsAuth.url}/health/ping`, config.apis.hmppsAuth.agent),
+  service('manageUsersApi', `${config.apis.manageUsersApi.url}/health/ping`, config.apis.manageUsersApi.agent),
```

Comment:
Ah, neat 👍 

So, not for this PR, but should we be putting an entry here for each of the API services that our UI is reliant on?
IE. our own API, Curious API, prisoner search, prison register etc ?